### PR TITLE
docs(python): port private registry docs to python

### PR DESCRIPTION
**Pull request by grouville**
- Port the private registry docs to python, to have a common basis
- Add it to guides list

It might be worth to have a better architecture other than copy-pasting ? I have several guides on-the-way that are worth being ported to python / go

cc @vikram-dagger 

Signed-off-by: Guillaume de Rouville

### CODE REVIEW

**Comment by vikram-dagger** [comment]
> It might be worth to have a better architecture other than copy-pasting ? I have several guides on-the-way that are worth being ported to python / go

You can use a partial for the common text and reuse it in each guide. Example of partial https://github.com/dagger/dagger/blob/main/docs/current/partials/_install-sdk-go.md. Example of usage https://github.com/dagger/dagger/blob/main/docs/current/sdk/go/959738-get-started.md?plain=1#L53

File: docs/current/sdk/python/guides/683293-private-repositories.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+```
+
+Now, first remove all the SSH keys from the authentication agent on the Dagger host:
+
+```shell
+➜  ssh-add -D
+All identities removed.
+```
+
+Attempt to run the Go CI tool:
```
Comment (vikram-dagger) [inline docs/current/sdk/python/guides/683293-private-repositories.md:24]:
```suggestion
Attempt to run the Python CI tool:
```

File: docs/current/sdk/python/snippets/private-repositories/clone.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# """
```
Comment (vikram-dagger) [inline docs/current/sdk/python/snippets/private-repositories/clone.py:1]: Not sure if the # is needed for these lines

File: docs/current/sdk/python/snippets/private-repositories/clone.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+import anyio
+import dagger
+
+async def private_repo():
+    async with dagger.Connection() as client:
+
+        repo = (
+            client.
```
Comment (vikram-dagger) [inline docs/current/sdk/python/snippets/private-repositories/clone.py:12]: I noticed that in @helderco examples, the `.` starts each line, whereas here it ends each line. Not sure what the standard convention is but we should be consistent in our code formatting.

**Review by helderco — APPROVED** [review]

File: docs/current/sdk/python/snippets/private-repositories/clone.py (hunk 3 above)
Comment (helderco) [reply docs/current/sdk/python/snippets/private-repositories/clone.py]: I think there's more a consensus on starting with the `.` in general, even in Go.

**Pull request merged**